Fix crossbeam-channel link & update example

twilligon opened the pull request:
This is a very trivial change, but it saves a couple clicks for every visitor. `crossbeam-channel` is now part of the main `crossbeam` repository, so link to its (`crossbeam-channel`'s) README instead of `crossbeam`'s.

twilligon committed:
Fix crossbeam-channel link

crossbeam-channel is now part of the main crossbeam repository, so link to its
README instead of that of the main repository. Saves two clicks ;)

ghost commented:
While we're at it, could you perhaps update the first example in the readme (the one with signals)? It was written in the v0.2 era when `send()` wasn't returning anything. Today, `send()` returns a `Result`.

So the suggestion is to change `s.send(signal);` to:
```rust
if s.send(signal).is_err() {
    break;
}
```

BurntSushi commented:
Looks good with @stjepang's suggestion!

milkey-mouse committed:
Handle error on send() in example

By @stjepang's suggestion: https://github.com/BurntSushi/chan-signal/pull/29#issuecomment-465289089

twilligon commented:
I added that change in a separate commit.

BurntSushi reviewed (approved):
Thanks!
